flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/job/SubtaskExecutionAttemptDetailsInfo.java

```diff
@@ -100,6 +106,36 @@ public class SubtaskExecutionAttemptDetailsInfo implements ResponseBody {
     @JsonProperty(FIELD_NAME_STATUS_DURATION)
     private final Map<ExecutionState, Long> statusDuration;
 
+    @JsonProperty(FIELD_NAME_OTHER_CONCURRENT_ATTEMPTS)
+    @JsonInclude(Include.NON_EMPTY)
+    @Nullable
+    private final List<SubtaskExecutionAttemptDetailsInfo> otherConcurrentAttempts;
+
+    public SubtaskExecutionAttemptDetailsInfo(
```

Comment:
Is this for test only? If so, I would suggest to remove it and refactor the tests instead.